CMakeLists.txt

```diff
@@ -700,6 +700,7 @@ foreach(f
   gtm_strings.h
   gtmgblstat.xc
   gtmxc_types.h
+  libyottadb.h
```

Comment:
This comment is actually a comment on the commit message placed here because I cannot comment on that anywhere else.

1) and -> but?
ZHALT no longer halts and returns its argument to the caller.

2) fixes -> issues?
Various fixes dealing with return values when HALT/ZHALT/ZGOTO 0 are unwinding the stack that exploded were fixed.

3) The same sentence in the above comment (2) could be better worded. It currently reads as : The stack explosion occured first and then when we unwind the stack we had other issues.

Reply:
1. Yes and done
2 & 3. Rewrote the paragraph. Now read:

    6. Fixes for various issues (mostly sig-11 segmentation violations) dealing with
       return values when HALT/ZHALT/ZGOTO 0 are unwinding the stack to return either
       a specified return value (ZHALT) or a default NULL or zero return value (HALT
       or ZGOTO 0).